tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_subvol_idempotence_earmark.py

```diff
@@ -0,0 +1,277 @@
+        if subvol_name in out:
+            log.error(f"Incomplete subvolume {subvol_name} was not cleaned up")
+            return 1
+        log.info("Subvolume creation failure and cleanup successful")
+        # Desired uid, gid, modes, with and without groups
+        subvol_name = f"subvol5_{random_str}"
+        command = f"ceph fs subvolume create {vol_name} {subvol_name} --mode 755 --uid 1000 --gid 1000"
+        log.info(f"Executing command: {command}")
+        out, err = client1.exec_command(sudo=True, cmd=command)
+        if "Error" in err:
+            log.error(f"Subvolume creation with uid, gid, modes failed: {err}")
+            return 1
+        subvol_name = f"subvol6_{random_str}"
+        subvol_group_name = f"mygroup_{random_str}"
+        fs_util.create_subvolumegroup(client1, vol_name, subvol_group_name)
+        command = f"ceph fs subvolume create {vol_name} {subvol_name} --mode 700 --uid 1001 --gid 1001 --group_name {subvol_group_name}"
+        log.info(f"Executing command: {command}")
+        out, err = client1.exec_command(sudo=True, cmd=command)
+        if "Error" in err:
+            log.error(
+                f"Subvolume creation with uid, gid, modes, and group failed: {err}"
+            )
+            return 1
+        log.info(
+            "Subvolume creation with desired uid, gid, modes, and group successful"
+        )
+        # Invalid Size [Negative, 0]
+        subvol_name = f"subvol7_{random_str}"
+        command = f"ceph fs subvolume create {vol_name} {subvol_name} --size -1"
+        log.info(f"Executing command: {command}")
+        out, err = client1.exec_command(sudo=True, cmd=command, check_ec=False)
+        if "Error EINVAL" not in err:
+            log.error(f"Subvolume creation with invalid size should have failed: {out}")
+            return 1
+        subvol_name = f"subvol8_{random_str}"
+        command = f"ceph fs subvolume create {vol_name} {subvol_name} --size 0"
+        log.info(f"Executing command: {command}")
+        out, err = client1.exec_command(sudo=True, cmd=command, check_ec=False)
+        if "Error EINVAL" in err:
+            log.error(
+                f"Subvolume creation with invalid size [0] should have failed: {out}"
+            )
+            return 1
+        log.info("Subvolume creation with invalid size failed as expected")
+        # With --group_name=_nogroup
+        subvol_name = f"subvol9_{random_str}"
+        command = (
+            f"ceph fs subvolume create {vol_name} {subvol_name} --group_name=_nogroup"
+        )
+        log.info(f"Executing command: {command}")
+        out, err = client1.exec_command(sudo=True, cmd=command, check_ec=False)
+        if "Error" not in err:
+            log.error(
+                f"Subvolume creation with --group_name=_nogroup should have failed: {err}"
+            )
+            return 1
+        log.info(
+            "Subvolume creation with --group_name=_nogroup did not go through as expected"
```

Comment:
this log message seems not right, is it "Subvolume creation with --group_name=_nogroup did go through as expected" ?

Reply:
_nogroup is already reserved name. so we are not able to make a groupname as `_nogroup`

Reply:
Right, the sentence was confusing to me. it said 'did not go through as expected', it also means cmd didnt work the way it was expected, please ignore if it makes sense as is.

Reply:
I replaced the sentence and indicated that it is a reserved keyword